src/components/dialogs/two-windings-transformer/ratio-tap-changer-pane.js

```diff
@@ -400,6 +390,7 @@ const RatioTapChangerPane = (props) => {
 
             tempRows.forEach((row, index) => {
                 tempRows[index].ratio = currentRatio;
+                tempRows[index].key = currentRatio;
```

Comment:
Add a comment here to show that you understand what you are doing

Reply:
done